src/Tizen.NUI.Components/Controls/Navigation/Page.cs

```diff
@@ -0,0 +1,118 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Binding;
+using System.Windows.Input;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// AppearingEventArgs is a class to record page appearing event arguments which will be sent to user.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class AppearingEventArgs : EventArgs
```

Comment:
Since it is not a nested class of `Page`, I think it'd better to specify what is appearing and disappearing.
> PageAppearingEventArgs
> PageDisappearingEventArgs

Reply:
I will change them as you commented.
Thank you.